css/game.css

```diff
@@ -135,6 +134,14 @@ img.missile {
     background-size: cover;
 }
 
+.explosion {
+    position: absolute;
+    bottom: -3%;
```

Comment:
why?

Reply:
id atleast do this with Px, and add a comment as to why

Reply:
This was done to account for blank space at the bottom of the gif. made pixels in 7ca9737